deploy/template.yaml

```diff
@@ -1,7 +1,7 @@
 AWSTemplateFormatVersion: "2010-09-09"
 Transform: AWS::Serverless-2016-10-31
 Description: >
-  di-devplatform-performance
+  di-ipv-cri-kiwi-performance
```

Comment:
Do we need to update the description in the CF template?

Reply:
Thanks - I'll revert these ones